packages/lodestar/src/chain/chain.ts

```diff
@@ -203,151 +156,7 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     });
     this.logger.info("Beacon chain initialized");
   }
-
-  public async isValidBlock(state: BeaconState, signedBlock: SignedBeaconBlock): Promise<boolean> {
-    // The parent block with root block.previous_block_root has been processed and accepted.
-    // const hasParent = await this.db.block.has(block.parentRoot);
-    // if (!hasParent) {
-    //   return false;
-    // }
-    // An Ethereum 1.0 block pointed to by the state.
-    // latest_eth1_data.block_hash has been processed and accepted.
-    // TODO: implement
-
-    return getCurrentSlot(this.config, state.genesisTime) >= signedBlock.message.slot;
-  }
-
-  private processBlock = async (job: IBlockProcessJob, blockHash: Root): Promise<void> => {
-    const parentBlock = await this.db.block.get(job.signedBlock.message.parentRoot.valueOf() as Uint8Array);
-    if (!parentBlock) {
-      this.logger.warn(`Block(${toHexString(blockHash)}) at slot ${job.signedBlock.message.slot} `
-          + ` is missing parent block (${toHexString(job.signedBlock.message.parentRoot)}).`
-      );
-      this.emit("unknownBlockRoot", job.signedBlock.message.parentRoot.valueOf() as Uint8Array);
-      setTimeout((queue) => queue.add(job), 1000, this.blockProcessingQueue);
-      return;
-    }
-    const pre = await this.db.state.get(parentBlock.message.stateRoot.valueOf() as Uint8Array);
-    const isValidBlock = await this.isValidBlock(pre, job.signedBlock);
-    assert(isValidBlock);
-    const hexBlockHash = toHexString(blockHash);
-    this.logger.info(`${hexBlockHash} is valid, running state transition...`);
-
-    // process current slot
-    const post = await this.runStateTransition(job.signedBlock, pre);
-
-    this.logger.info(
-      `Slot ${job.signedBlock.message.slot} Block ${hexBlockHash} ` +
-      `State ${toHexString(this.config.types.BeaconState.hashTreeRoot(post))} passed state transition`
-    );
```

Comment:
Seems like we miss the below operations. Added back in https://github.com/ChainSafe/lodestar/pull/806